extensions/admin-tool-ext-win/package.json

```diff
@@ -70,7 +70,7 @@
 		]
 	},
 	"dependencies": {
-		"vscode-extension-telemetry": "^0.0.15",
+		"ads-extension-telemetry": "github:Charles-Gagnon/ads-extension-telemetry#0.1.0",
```

Comment:
This is in my repo for now but I'll be moving it to the MS repo ASAP - just wanted to get this out first before going through the OSS process.